container/webui/run_openqa.sh

```diff
@@ -40,6 +41,7 @@ function all_together_apache() {
 }
 
 usermod --shell /bin/sh geekotest
+chown -R geekotest /data    
```

Comment:
This directory is shared between (composed) containers, right? Do we somehow ensure that `geekotest` always has the same underlying user ID across all (composed) containers? I suppose this would be required for this line to work.

Note that the spec file alone does *not* seem to ensure that:

```
%pre
if ! getent passwd geekotest > /dev/null; then
    %{_sbindir}/useradd -r -g nogroup -c "openQA user" \
        -d %{_localstatedir}/lib/openqa geekotest 2>/dev/null || :
fi
```

You could of course extend the `usermod` command above:

```
usermod -u 2005 --shell /bin/sh geekotest
```

(I haven't tested the usermod command and it is still a bit hacky since it relies on 2005 not being already used.)

Reply:
I don't think we want to hardcode an expected UID in a spec file

Reply:
I would also not hard-code the UID in the spec file (but possibly hard-code it in the container script as suggested unless we have a better idea).

Btw, I've checked some systems with openQA packages installed and the UID indeed varies a little bit between them. However, maybe that's not an issue here because we just spawn multiple instances of the very same container image?